src/lib/viewers/doc/DocFirstPreloader.js

```diff
@@ -436,6 +437,73 @@ class DocFirstPreloader extends EventEmitter {
         });
     }
 
+    readEXIFNew(imageBlob, imageEl) {
```

Comment:
Is the old `readEXIF` function still used? Why don't we remove it and replace it with this one?

Reply:
Good catch